css/style.css

```diff
@@ -0,0 +1,174 @@
+body {
+    font-family: 'Poppins', sans-serif;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.4;
+    margin: 0;
+    background-color:rgb(6, 52, 36);
+    color: white;
+  }
+  
+  
+  h1 {
+    font-weight: 400;
+    line-height: 1.2;
+  }
+  
+  p {
+    font-size: 1.125rem;
+  }
+  
+  h1,
+  p {
+    margin-top: 0;
+    margin-bottom: 0.5rem;
+  }
+  
+
+  label {
+    display: flex;
+    align-items: center;
+    font-size: 1.125rem;
+    margin-bottom: 0.5rem;
+  }
+  
+ 
+  textarea {
+    margin: 0;
+    font-family: inherit;
+    font-size: inherit;
+    line-height: inherit;
+  }
+  input{
+    margin: 0;
+    font-family: inherit;
+    font-size: inherit;
+    line-height: inherit;
+  }
+
+  button{
+    margin: 0;
+    font-family: inherit;
+    font-size: inherit;
+    line-height: inherit;
+
+  }
+
```

Comment:
- [ ]  I can see that you have repeated similar styles on different HTML tags, I would recommend you to use the DRY ( Don't Repeat Yourself ) method. For example, you can improve the highlighted codes  in this way; 
![codesnap](https://user-images.githubusercontent.com/43877689/169037586-78c77607-dc1d-488a-aab6-e56c4af43354.png)

Reply:
- [ ] You have to use an optional keyword before optional comments like this
`- [ optional] your optional comment comment here`